src/components/Markdown.react.js

```diff
@@ -13,7 +13,7 @@ function DashMarkdown(props) {
         props.children = props.children.join('\n');
     }
 
-    return <Markdown source={props.children} escapeHtml={true} {...props} />;
+    return <Markdown source={props.children} {...props} />;
```

Comment:
I think you're missing something like `escapeHtml={props.dangerouslyEscapeHtml}` here - the `{...props}` spread here isn't enough because of the naming difference (dangerouslyEscapeHtml vs escapeHtml). Could you add that?

Reply:
Thanks for pointing this out, I forgot. Once I resolve the naming/boolean question below, it'll be in the next commit.